src/coldfront_plugin_api/scim_v2/groups.py

```diff
@@ -95,6 +108,10 @@ def patch(self, request, pk, format=None):
                     signals.allocation_remove_user.send(
                         sender=self.__class__, allocation_user_pk=au.pk,
                     )
+
+                    self._set_user_on_project(
+                        project, user, "Removed", "User", False
```

Comment:
Do not remove a user from the project. They might be in other resource allocations of the project.